src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -30,6 +30,7 @@ Multiple projects provide implementations of sidechains that you can integrate i
 - [Polygon PoS](https://polygon.technology/solutions/polygon-pos)
 - [Skale](https://skale.network/)
 - [Gnosis Chain (formerly xDai)](https://www.xdaichain.com/)
+- [Binance Smart Chain](https://academy.binance.com/en/articles/how-to-get-started-with-binance-smart-chain-bsc)
```

Comment:
The definition of a sidechain might not make this totally clear, but I don’t think BSC is an Ethereum sidechain.
While it is EVM compatible and bridges to Ethereum, it is not running in parallel to Ethereum and uses a different token standard (ERC20 vs. BEP2).
If anything, it is more a sidechain to the Binance Chain than to Ethereum. The Binance Smart Chain whitepaper is pretty clear on the fact that BSC runs in parallel to the Binance Chain and that Binance Chain is the mainnet it’s connected with, not Ethereum: 
“_After the creation of the parallel blockchain into the Binance Chain ecosystem, two blockchains will run side by side to provide different services. The new parallel chain will be called “Binance Smart Chain” (short as “BSC” for the below sections), while the existing mainnet remains named “Binance Chain” (short as “BC” for the below sections)._”
Link to whitepaper: https://github.com/bnb-chain/whitepaper/blob/master/WHITEPAPER.md

Reply:
I think given the context that this a sidechain in the Binance Chain ecosystem with its own token standards, it would be a bit odd to put it on the ethereum.org website. While it is technically a sidechain, it does feel out of place in the context of this page.